### PR TITLE
premake and appveyor update

### PR DESCRIPTION
**Pull request by ThirteenAG**
- cleaned up configurations
- added .gitignore
- fixed compilation error without installed mfc
- copy postbuild event doesn't fail a build and only copies output file if it exists in second directory

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
 environment:
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
 install:
-  - set PATH=%APPVEYOR_BUILD_FOLDER%/bin;%PATH%
-before_build:
-  - mkdir "%APPVEYOR_BUILD_FOLDER%/build"
-  - cd "%APPVEYOR_BUILD_FOLDER%"
-  - premake5.exe vs2015
+- cmd: >-
+    git submodule update --init --recursive
```
Comment (ShFil119) [inline .appveyor.yml]: Is it needed?

File: .appveyor.yml [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+    premake-vs2019.cmd
 build:
-  project: c:\projects\re3\build\re3.sln
+  project: build/re3.sln
   verbosity: minimal
-artifacts:
-  path: bin/%CONFIGURATION%/re3.dll
+after_build:
+- ps: >-
+    $releases = "https://github.com/ThirteenAG/Ultimate-ASI-Loader/releases"
```
Comment (ShFil119) [inline .appveyor.yml]: I'd add comment what's happening here. Also remove unneeded empty lines below.

File: .appveyor.yml (hunk 1 above)
Comment (ThirteenAG) [reply .appveyor.yml]: Not at the moment, no, but I figured it can't hurt to have it, in case dxsdk/rwsdk will be moved to submodule or something.

**Comment by ShFil119** [comment]
Also maybe you should update binary of premake.

**Comment by ThirteenAG** [comment]
>Also maybe you should update binary of premake.

Did that, seems like this repo already contains the latest version to date.

File: .appveyor.yml (hunk 2 above)
Comment (ThirteenAG) [reply .appveyor.yml]: That's how appveyor exports this file, I don't add empty lines specifically.

**Pull request merged**